### ISSUE TITLE
Flag for changing letter-spacing 

### ISSUE THREAD

**Issue by rajeshvaya** [Microsoft/vscode#18715]
- VSCode Version: 1.7.2
- OS Version: Mac OSX

**Comment by dbaeumer** [comment]
@rajeshvaya can you be a little bit more specific. Should that change in the editor, overall?

**Comment by rajeshvaya** [comment]
@dbaeumer, like we have the following settings for changing fonts and line spacing

```
"editor.lineHeight": 24,
 "editor.insertSpaces": true,
 "editor.fontSize": 15
```

It will be good if we have another flag / setting as `editor.letterSpacing: XX`  to configure character spacing in the editor. (something similar to CSS letter-spacing property)

**Comment by alexandrudima** [comment]
Implemented via PR #22979 

Thank you @hoovercj
